mqtt/src/test/scala/docs/scaladsl/MqttFlowSpec.scala

```diff
@@ -64,5 +67,42 @@ class MqttFlowSpec
       mqttMessagePromise.success(None)
       noException should be thrownBy result.futureValue
     }
+    "send an ack after sent confirmation" in {
+      val topic = "flow-spec-ack/topic"
+      //#create-flow
```

Comment:
You need to remove these markers, they are used to pull snippets into the docs.

Reply:
> Ok, this looks good now. Remaining things:
> 
> * [ ]  I don't understand the use case of `MqttSource.atLeastOnceWithAck`.
> * [ ]  Add the factory methods in the Java DSL
> * [ ]  Documentation

MqttSource.atLeastOnceWithAck it's not correct because i touched only the flow..
i remove this method in the next commit...
at the moment i'm searching the problem of the tests.. seems that sometimes ( that i don't understand yet ) the method 'delivery Complete ' is not called.
for the second tick, what documentation do you mean?

thank you for your patience in my first P.R  :)